### PR TITLE
Refactor capacity component

### PR DESCRIPTION
**Pull request by Keith-CY**
This PR refactors the `Capacity` component
image

Affected pages are as follows:
1. Homepage: reward of a block, capacity of a transaction, https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/
2. Block list: reward of a block, https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/block/list
3. Transaction list: capacity of a transaction, https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/transaction/list
4. Transaction detail(professional/lite): capacity of each cell, https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/transaction/0xc0dba027b4fe745b7f520bc87cbf8d77ac8a0b0a501e773db0c2eecc6ece817a
5. Address detail(professional/lite): capacity of each cell, https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/address/ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqgy2q6qz79wyaexr2pcez0eejmk5xgw6jcfw7zmg
6. Nervos DAO: distribution in overview, capacity of transaction, deposited capacity of an address, popup of nervos dao icon, https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/nervosdao
7. Script detail: capacity of transaction, capacities in overview, https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/script/0x79f90bb5e892d80dd213439eeab551120eb417678824f282b4ffb5f21bad2e1e/type/deployed_cells?page=1&size=10

### CODE REVIEW

File: src/components/Capacity/Capacity.module.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      }
+    }
+
+    &[data-diff-status='negative'] {
+      color: var(--accent-color);
+    }
+  }
+
+  &[data-layout='responsive'] {
+    // FIXME: use css variable later
```
Comment (WhiteMinds) [inline src/components/Capacity/Capacity.module.scss:35]: It is now ready to use because #178 has been merged

File: src/pages/Home/TableCard/index.module.scss [hunk 2]
```diff
@@ -13,3 +13,44 @@
...
+    font-size: 9px;
+    padding-right: 2px;
+  }
+
+  span[data-role='suffix'] {
+    font-size: 13px;
+  }
+
+  // FIXME: use css variable later
+  @media screen and (width<=1000px) {
```
Comment (WhiteMinds) [inline src/pages/Home/TableCard/index.module.scss:33]: This line is not formatted

**Review by WhiteMinds — APPROVED** [review]

**Comment by WhiteMinds** [comment]
This PR is based on #180, and now we need to remove unnecessary changes from #180.

**Comment by Keith-CY** [comment]
> This PR is based on #180, and now we need to remove unnecessary changes from #180.

Updated and scss variables are used in media queries

File: src/components/TransactionItem/TransactionIncome/styled.tsx [hunk 3]
```diff
@@ -22,6 +22,9 @@ export const TransactionCapacityValuePanel = styled.div<{ increased: boolean }>`
   justify-content: flex-end;
   color: ${props => (props.increased ? props.theme.primary : '#FF6347')};
   font-size: 16px;
+  .monospace {
+    color: inherit;
+  }
```
Comment (WhiteMinds) [inline src/components/TransactionItem/TransactionIncome/styled.tsx:27]: This commit has not been removed.

**Review by WhiteMinds — CHANGES_REQUESTED** [review]

File: src/components/TransactionItem/TransactionIncome/styled.tsx (hunk 3 above)
Comment (Keith-CY) [reply src/components/TransactionItem/TransactionIncome/styled.tsx]: Redundant style was removed by https://github.com/Magickbase/ckb-explorer-frontend/pull/181/commits/85fbdad61888c938eabea380d8960be06e1393db

**Review by WhiteMinds — APPROVED** [review]

**Comment by FrederLu** [comment]
UI:
image

1、The color of the data `()` in the transaction page Lite is abnormal.
https://ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0x1d320432dbe19599eb752d835be9877a596870b46980dd68edad3abd1fd10111?layout=lite
image

---
2、The id information style and `()` color are inconsistent with before.
https://ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0xb730c4ddaf41272532a809383b45731de990971bdee9eecfbe22fd5af816fdb1?layout=lite
image

Previous style:
image

---
3、When there is a label, the distance between it and the following information is small and needs to be adjusted.
https://ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0x7ab270c95cb31d1ee32b1b56433248c0a73bb462536a20df873122f8b6bfa7e1?layout=lite
image

**Comment by Keith-CY** [comment]
> UI: image
> 
> 1、The color of the data `()` in the transaction page Lite is abnormal. [ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0x1d320432dbe19599eb752d835be9877a596870b46980dd68edad3abd1fd10111?layout=lite](https://ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0x1d320432dbe19599eb752d835be9877a596870b46980dd68edad3abd1fd10111?layout=lite) image
> 
> 2、The id information style and `()` color are inconsistent with before. [ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0xb730c4ddaf41272532a809383b45731de990971bdee9eecfbe22fd5af816fdb1?layout=lite](https://ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0xb730c4ddaf41272532a809383b45731de990971bdee9eecfbe22fd5af816fdb1?layout=lite) image
> 
> Previous style: image
> 
> 3、When there is a label, the distance between it and the following information is small and needs to be adjusted. [ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0x7ab270c95cb31d1ee32b1b56433248c0a73bb462536a20df873122f8b6bfa7e1?layout=lite](https://ckb-explorer-frontend-in-magickbase-repo-r61x41klx-magickbase.vercel.app/transaction/0x7ab270c95cb31d1ee32b1b56433248c0a73bb462536a20df873122f8b6bfa7e1?layout=lite) image

The color of NFT ID and the distance between label and amount will be fixed.

The color of `()` need more detail because in previous design, the `A` and `B` in `A(B)` wre assumed to be the same, so the `()` has the same color. But in fact `A` and `B` can be colored differently, so the color of `()` should be specified to be the same as `A` or `B` cc @Sven-TBD

**Comment by Keith-CY** [comment]
Ah, I recalled that color of NFT is not set because the direction of NFT is not set, as mentioned at https://github.com/Magickbase/ckb-explorer-public-issues/issues/503

They are previously colored because the assumption that `NFT` is sent when the capacity is negative, or `NFT` is received when the capacity is positive. But the assumption is invalid.

So there need more detail returned from API to set the color of NFT ID

**Comment by FrederLu** [comment]
>The color of NFT ID and the distance between label and amount will be fixed.
The color of () need more detail because in previous design, the A and B in A(B) wre assumed to be the same, so the () has the same color. But in fact A and B can be colored differently, so the color of () should be specified to be the same as A or B cc @Sven-TBD

>Ah, I recalled that color of NFT is not set because the direction of NFT is not set, as mentioned at https://github.com/Magickbase/ckb-explorer-public-issues/issues/503
They are previously colored because the assumption that NFT is sent when the capacity is negative, or NFT is received when the capacity is positive. But the assumption is invalid.
So there need more detail returned from API to set the color of NFT ID cc @FrederLu

To sum up, the label position can now be fine-tuned, and the rest needs to be processed later.

**Comment by Keith-CY** [comment]
> > The color of NFT ID and the distance between label and amount will be fixed.
> > The color of () need more detail because in previous design, the A and B in A(B) wre assumed to be the same, so the () has the same color. But in fact A and B can be colored differently, so the color of () should be specified to be the same as A or B cc @Sven-TBD
> 
> > Ah, I recalled that color of NFT is not set because the direction of NFT is not set, as mentioned at [Magickbase/ckb-explorer-public-issues#503](https://github.com/Magickbase/ckb-explorer-public-issues/issues/503)
> > They are previously colored because the assumption that NFT is sent when the capacity is negative, or NFT is received when the capacity is positive. But the assumption is invalid.
> > So there need more detail returned from API to set the color of NFT ID cc @FrederLu
> 
> To sum up, the label position can now be fine-tuned, and the rest needs to be processed later.

Yes

**Comment by FrederLu** [comment]
> > > The color of NFT ID and the distance between label and amount will be fixed.
> > > The color of () need more detail because in previous design, the A and B in A(B) wre assumed to be the same, so the () has the same color. But in fact A and B can be colored differently, so the color of () should be specified to be the same as A or B cc @Sven-TBD
> > 
> > 
> > > Ah, I recalled that color of NFT is not set because the direction of NFT is not set, as mentioned at [Magickbase/ckb-explorer-public-issues#503](https://github.com/Magickbase/ckb-explorer-public-issues/issues/503)
> > > They are previously colored because the assumption that NFT is sent when the capacity is negative, or NFT is received when the capacity is positive. But the assumption is invalid.
> > > So there need more detail returned from API to set the color of NFT ID cc @FrederLu
> > 
> > 
> > To sum up, the label position can now be fine-tuned, and the rest needs to be processed later.
> 
> Yes

OK.  The style here will be different from the previous design. cc @Sven-TBD

**Comment by Keith-CY** [comment]
This distance between transfer type and amount should be fixed by https://github.com/Magickbase/ckb-explorer-frontend/pull/181/commits/1c06b16ff71113690499aa771d6eef04f8c0ad6b
Preview is available at https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/transaction/0x7ab270c95cb31d1ee32b1b56433248c0a73bb462536a20df873122f8b6bfa7e1?layout=lite

**Review by FrederLu — APPROVED** [review]
